js/content/community.js

```diff
@@ -291,6 +292,260 @@ let CommentHandler = (function(){
         observer.observe(document.body, { childList: true });
     };
 
+    function loadScripts() {
+        return new Promise(async function(res) {
```

Comment:
As above, `res` is shorthand for `resolve`. type the extra 4 characters. If any of the scripts fail to load we should still let users use the un-upgraded text box and the other AS features, which means you need to catch and handle errors here.

Why create a new Promise instead of using the existing `loadInPageContext()` promises, or a `Promise.all()` to collect them? OR just make `loadScripts()` `async` and not have any mention of `Promise` here at all?

Reply:
As a tip, the `async` keyword means the function returns a Promise, and wrapping a function that returns a Promise in a `new Promise()` is redundant.

Reply:
> As a tip, the `async` keyword means the function returns a Promise, and wrapping a function that returns a Promise in a `new Promise()` is redundant.

I did not know this. I though async was solely to support `await`.